backend/app/extraction/ExternalTranscriptionExtractor.scala

```diff
@@ -0,0 +1,126 @@
+package extraction
+
+import com.amazonaws.services.sqs.AmazonSQS
+import com.amazonaws.services.sqs.model.SendMessageRequest
+import model.manifest.Blob
+import org.joda.time.DateTime
+import play.api.libs.json.Json
+import services.index.Index
+import services.{ObjectStorage, TranscribeConfig}
+import utils._
+import utils.attempt.Failure
+
+import java.util.UUID
+import scala.concurrent.ExecutionContext
+
+case class SignedUrl(url: String, key: String)
+
+object SignedUrl {
+  implicit val formats = Json.format[SignedUrl]
+}
+case class OutputBucketUrls(text: SignedUrl, srt: SignedUrl, json: SignedUrl)
+case class OutputBucketKeys(text: String, srt: String, json: String)
+case class TranscriptionJob(id: String, originalFilename: String, inputSignedUrl: String, sentTimestamp: String,
+                            userEmail: String, transcriptDestinationService: String, outputBucketUrls: OutputBucketUrls,
+                            languageCode: String, translate: Boolean)
+object OutputBucketUrls {
+  implicit val formats = Json.format[OutputBucketUrls]
+}
+
+object OutputBucketKeys {
+  implicit val formats = Json.format[OutputBucketKeys]
+}
+object TranscriptionJob {
+  implicit val formats = Json.format[TranscriptionJob]
+}
+
+/**
+  * id: z.string(),
+  * originalFilename: z.string(),
+  * userEmail: z.string(),
+  * status: z.literal('SUCCESS'),
+  * languageCode: z.string(),
+  * outputBucketKeys: OutputBucketKeys,
+  */
```

Comment:
this could probably be removed - maybe replace with a comment linking to the relevant `types.ts` file in the transcription-service repo